extensions/tika/runtime/pom.xml

```diff
@@ -39,6 +39,20 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>ooxml-schemas</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.xmlbeans</groupId>
+                    <artifactId>xmlbeans</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
You can remove this exclusion and the dependency below (since it's resolved transitively)

Reply:
Hi @gastaldi - There was a difference in the versions of XML Beans. I found only this way to deal with it.